research/H1/AAL H1 message

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Can you extract the details from this so that it's not just a JSON dump of the Airframes decoder output? Some non-"programmer" formatting would be helpful here.

Reply:
Yes, I will work on that.

Reply:
Actually, that file 'AAL H1 message' was not intended to be part of my submission.  It was just a scratch file I was saving that message in.